tslint.json

```diff
@@ -0,0 +1,100 @@
+{
+    "defaultSeverity": "error",
+    "extends": "typestrict",
+    "jsRules": {},
+    "rules": {
+        "arrow-parens": true,
+        "ban-ts-ignore": true,
+        "import-spacing": true,
+        "indent": {
+            "options": [
+                "spaces",
+                2
+            ]
+        },
+        "max-line-length": {
+            "options": {
+                "limit": 120
+            }
+        },
+        "no-debugger": true,
+        "no-console": true,
+        "new-parens": true,
+        "only-arrow-functions": {
+            "options": [
+                "allow-declarations",
+                "allow-named-functions"
+            ]
+        },
+        "no-consecutive-blank-lines": [
+            true,
+            2
+        ],
+        "no-irregular-whitespace": true,
+        "no-trailing-whitespace": {
+            "options": [
+                "ignore-template-strings"
+            ]
+        },
+        "no-unused-variable": false, // disabled because deprecated
```

Comment:
This seems to be enabled by typestrict, however tslint actually warns that `no-unused-variable` is depreciated since the typescript compiler now has options for `"noUnusedLocals"` and `"noUnusedParameters"` (which I've enabled).

Note: Github is complaining about the comments in this `.json` file but tslint is cool with it.

Reply:
Let's remove the comment. The Git history can serve as a comment. 🙂